src/bandersnatch/tests/plugins/test_filename.py

```diff
@@ -159,4 +189,4 @@ def test_exclude_platform(self) -> None:
         assert sum(f["flag"] == "DROP" for r in rv for f in r) == 0
 
         # the release "0.2" should have been deleted since there is no more file in it
-        assert len(pkg.releases.keys()) == 2
+        assert len(pkg.releases.keys()) == 3
```

Comment:
Why is this 3 now? Maybe add that to the comment.

Reply:
I added another 'release' in the test data - lines 150 ~ 176 - that shouldn't be filtered out.
The test still checks the count of 'KEEP's and 'DROP's, which I think is sufficient for the new filtering logic.